core/test/lib/tracehouse/trace-processor-test.js

```diff
@@ -217,14 +216,21 @@ describe('TraceProcessor', () => {
         /* eslint-enable max-len */
       );
       const trace = TraceProcessor.processTrace(testTrace);
-      expect(trace.frameTreeEvents.map(e => e.name)).toEqual([
-        'navigationStart',
-        'domContentLoadedEventEnd',
-        'firstContentfulPaint',
-        'firstMeaningfulPaint',
-        'Event1',
-        'Event2',
-      ]);
+      const frameTreeEventOutput = trace.frameTreeEvents.map((e) =>
+        `${e.name.padEnd(25)} - ${e.args?.data?.frame || e.args.frame}`);
+
+      expect(frameTreeEventOutput).toMatchInlineSnapshot(`
+        Array [
+          "navigationStart           - ROOT_FRAME",
+          "FrameCommittedInBrowser   - ROOT_FRAME",
```

Comment:
diff-wise: the two `FrameCommittedInBrowser` items here are the new rows. 
I added the frame identifier to spell out the tree.